lib/frame.js

```diff
@@ -596,35 +596,30 @@ function _filterSubject(state, subject, frame, flags) {
           return false;
         }
         matchThis = true;
-      } else if(types.isObject(thisFrame)) { // XXX only framing keywords
-        // node matches if values is not empty and the value of property in
-        // frame is wildcard
-        matchThis = nodeValues.length > 0;
       } else {
-        if(graphTypes.isValue(thisFrame)) {
-          // match on any matching value
-          matchThis = nodeValues.some(nv => _valueMatch(thisFrame, nv));
-        } else if(graphTypes.isSubject(thisFrame) ||
-          graphTypes.isSubjectReference(thisFrame)) {
-          matchThis =
-            nodeValues.some(nv => _nodeMatch(state, thisFrame, nv, flags));
-        } else if(graphTypes.isList(thisFrame)) {
+        if (graphTypes.isList(thisFrame)) {
```

Comment:
Please run "npm run lint".  Style we use omits space after `if`.

Reply:
OK, should be resolved now.